tests/python/test_views.py

```diff
@@ -35,11 +38,14 @@ def setUp(self):
             self.run = dict(res.fetchone())
 
     def tearDown(self):
-        with tables(db.engine, 'projects', 'bams', 'vcfs') as (con, projects, bams, runs):
+        with tables(db.engine, 'projects', 'bams', 'vcfs', 'user_comments') \
```

Comment:
Ah yes you did on the other test PR; I'll do it on this one, and then take care of porting it to the other tests after these PRs are merged in?